scripts/run-framework/agief_experiment/cloud.py

```diff
@@ -101,7 +101,7 @@ def ecs_stop_task(self, task_arn):
         )
 
         if self.log:
```

Comment:
Since we're using proper logging, we can now remove the `if self.log:` and all other places where we are currently manually keeping track of log level. That means also removing it as a parameter from the methods.